### PR TITLE
Add verify_different_msg to aggregate signatures

### PR DESCRIPTION
**Pull request by jonas-lj**
Closing #98

### CODE REVIEW

File: fastcrypto/src/ed25519.rs [hunk 1]
```diff
@@ -444,6 +444,26 @@ impl AggregateAuthenticator for Ed25519AggregateSignature {
             .map_err(|_| FastCryptoError::GeneralError)
     }
 
+    fn verify_different_msg(
```
Comment (kchalkias) [inline fastcrypto/src/ed25519.rs:447]: what's the difference between this and `batch_verify`?

Comment (jonas-lj) [reply fastcrypto/src/ed25519.rs:447]: As I understand it, `batch_verify`verifies a batch of aggregate signatures, where each aggregate signature is on the same message, but `verify_different_msg`verifies a single aggregate signature different messages.

Comment (kchalkias) [reply fastcrypto/src/ed25519.rs:447]: Ok will approve now and we can work on the ideal api later.

**Review by kchalkias — APPROVED** [review]

File: fastcrypto/src/ed25519.rs (hunk 1 above)
Comment (jonas-lj) [reply fastcrypto/src/ed25519.rs:447]: Great. To keep consistency, we might also want to add `batch_verify_different_msg`?

**Pull request merged**